packages/react-storage/src/components/StorageBrowser/Views/LocationActionView/UploadControls.tsx

```diff
@@ -69,6 +70,15 @@ export const ActionIcon = ({ status }: ActionIconProps): React.JSX.Element => {
   );
 };
 
+const Destination = ({ children }: { children?: React.ReactNode }) => {
+  return (
+    <DefinitionList className="storage-browser__destination">
+      <DefinitionTerm>Destination:</DefinitionTerm>
+      <DefinitionDetail>{children}</DefinitionDetail>
+    </DefinitionList>
+  );
+};
```

Comment:
Don't think we need a list for a single element

Reply:
Semantically it needs to be in a `dl` to use the other elements. It's fine to have a single definition.

Reply:
Got it